external/io_grib_share/build/library_rules.mk

```diff
@@ -148,7 +148,7 @@ thisdir_clean: thisdir_clean_lib
 	@/bin/rm -f *.o *.obj *.mod *.f90 core so_locations Makefile.bak *~ #*#
 	@/bin/rm -fr ii_files
 	@if [ -d utest ] ; then \
-		echo "        Doing make clean on utest subdirectory"; \
+#		echo "        Doing make clean on utest subdirectory"; \
```

Comment:
@davegill 
Same with this one - should we just remove the line altogether?

Reply:
Done